packages/components/src/components/scroll-buttons/test.html

```diff
@@ -18,7 +18,7 @@
 </head>
 
 <body>
-    <section style="display: flex;">
+    <section style="display: flex; background-color: #eee;">
```

Comment:
Do we need this change here? 🤔

Reply:
For internal testing. The buttons are now white, which makes them hard to see on a white background.